### ISSUE TITLE
[1.7.2] "Duplication" issue when breaking microblocks (FMP) with TC tools.

### ISSUE THREAD

**Issue by dowttie** [SlimeKnights/TinkersConstruct#688]
Whenever you have various microblocks within one block space and try to break them what would usually happen is that you'd break one of the microblocks at a time also getting one in item form at a time, but when I use TC tools, eg a TC pickaxe on microblocks you get all the microblocks at once while you actually only broke one. So eg, when you have 7 microblocks in one block space and you break one with a TC tools, 7 are dropped in item form, break the second one, you receive 6... and so one and so forth.
In short, when you break microblocks you receive the whole content of the block each time, even though only a microblock should've been dropped.

Tested with TC 1.6.0a14 mantle 0.2.8 forge 1110 & FMP 280.
